recipes/python/patches/ctypes-find-library.patch

```diff
@@ -0,0 +1,21 @@
+--- Python-2.7.2.orig/Lib/ctypes/util.py	2011-06-11 16:46:24.000000000 +0100
++++ Python-2.7.2/Lib/ctypes/util.py	2015-03-26 13:45:33.322172603 +0000
+@@ -83,7 +83,17 @@
+             except ValueError:
+                 continue
+         return None
+-
++elif os.name == "posix" and sys.platform == "linux3":
```

Comment:
This is not a good way to do this. It works right now, while we use Python 2.7.2. Python 2.7.3+ will always report `linux2` regardless of the Linux version, and Python 3 dropped the number entirely. You should just replace the original function rather than adding extra checks, since we want them to always evaluate to True anyway. :)